doc/nrf/libraries.rst

```diff
@@ -32,6 +32,11 @@ Here you can find documentation for these libraries, including API documentation
 
    ../../include/net/*
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Libraries for networking:
+   :glob:
+   ../../include/net/zzhc/*
```

Comment:
Please add a blank line before this line for this section to appear on the Libraries page.

Reply:
Added and pushed.